spotify_project/testing/base.py

```diff
@@ -36,3 +40,34 @@ def assert_num_objects_created(self, counts):
         for Model in counts:
             delta = after_counts[Model] - before_counts[Model]
             self.assertEqual(delta, counts[Model], Model)
+
+    @property
+    def dummy_library_data(self):
+        with open('../track_response_example.json') as json_file:
```

Comment:
This file does not seem to be in source control. Accident?

Reply:
Looks like I was wrong, my bad.

Reply:
No. I added it after your comment.